src/test/kotlin/org/rust/ide/intentions/RsIntentionTestBase.kt

```diff
@@ -11,9 +11,27 @@ import com.intellij.util.ui.UIUtil
 import org.intellij.lang.annotations.Language
 import org.rust.RsTestBase
 import org.rust.fileTreeFromText
+import java.nio.file.Paths
 
 abstract class RsIntentionTestBase(val intention: IntentionAction) : RsTestBase() {
 
+    fun testIntentionHasDocumentation() {
+        val directory = "intentionDescriptions/${intention.javaClass.simpleName}"
+
+        val description = Paths.get(directory, "description.html")
+        val text = getResourceAsString(description.toString())
+            ?: error("No intention description for ${intention.javaClass} ($description)")
+        checkHtmlStyle(text)
+
+        val afterTemplatePath = Paths.get(directory, "after.rs.template").toString()
+        getResourceAsString(afterTemplatePath)
+            ?: error("No after.rs.template found for ${intention.javaClass} ($afterTemplatePath)")
+
+        val beforeTemplatePath = Paths.get(directory, "before.rs.template").toString()
+        getResourceAsString(beforeTemplatePath)
+            ?: error("No before.rs.template found for ${intention.javaClass} ($beforeTemplatePath)")
```

Comment:
We use this similar code three times in a row. Maybe it's worth to extract it into separate method?